test_extras/testrunners.py

```diff
@@ -127,14 +129,15 @@ def get_coverage_files(app_labels, ignore_dirs, ignore_files):
     return a list of all the python files that should be included in
     a coverage report for that test.
     """
-    from django.db.models import get_app
-
     ret = []
 
     for app_label in app_labels:
-        module = get_app(app_label)
-        dirname = os.path.dirname(module.__file__)
-        ret.extend(get_coverage_files_in_directory(dirname, ignore_dirs, ignore_files))
+        if app_label in settings.INSTALLED_APPS:
+            module = __import__(app_label, fromlist=[''])
```

Comment:
I've updated the __import__call to include globals() and locals(). You will probably see that I'm not using the [] in the call. That's because in case of dotted imports, using [] instead of [''] returns the base module.